src/components/DeleteButton.spec.js

```diff
@@ -0,0 +1,29 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import DeleteButton from './DeleteButton.js';
+
+describe('DeleteButton', () => {
+  it('renders a button, svg and sr-only text', () => {
+    const deleteClick = jest.fn();
```

Comment:
probably you don't need the mock function, in case you just want to test the rendering

Reply:
thx 👍